project/Http4sPlugin.scala

```diff
@@ -104,7 +104,7 @@ object Http4sPlugin extends AutoPlugin {
     val cats = "2.11.0"
     val catsEffect = "3.5.4"
     val catsParse = "1.0.0"
-    val circe = "0.14.8"
+    val circe = "0.14.9"
```

Comment:
> Scala native 0.4 has been dropped in favor of 0.5 in https://github.com/circe/circe/pull/2254

Reply:
That's an example of a library in the same place as us (#7474): early semver, and too foundational to break without massive ecosystem pain.